docs/php.md

```diff
@@ -14,6 +14,17 @@ xdebug.start_with_request=yes
 xdebug.log_level = 0
 ```
 
+The following configuration can be useful also to debug from the command line:
+
+```ini
+[xdebug]
```

Comment:
@geekdenz is there any difference to the ini above re CLI? It's the same except for also having the extension in there and a higher max nesting level.

Reply:
Thanks for pointing this out. Is it better now?